weave-js/src/common/components/elements/ModifiedDropdown.tsx

```diff
@@ -206,8 +206,20 @@ const ModifiedDropdown: FC<ModifiedDropdownProps> = React.memo(
       return itemCount() >= itemLimit;
     }, [itemLimit, itemCount]);
 
+    const computedOptions = searchQuery ? options : propsOptions;
     const displayOptions = getDisplayOptions(
-      searchQuery ? options : propsOptions,
+      multiple
+        ? computedOptions
+        : computedOptions.map(opt => ({
+            ...opt,
+            content: (
+              <div
+                style={{padding: '13px 18px', margin: '-13px -18px'}}
+                onClick={() => setSearchQuery('')}>
+                {opt.text}
+              </div>
+            ),
```

Comment:
Semantic UI doesn't support `onSelect` or call `onChange` when the same selection is selected. The tests seem to break when using `onClose` and the other associated events like `onBlur` don't meet the ticket criteria. Because of this a custom option content is required.